frontend/packages/console-shared/src/utils/pod-ring-utils.ts

```diff
@@ -25,31 +24,30 @@ const applyPods = (podsData: PodRingData, dc: PodRCData) => {
 
 export const transformPodRingData = (resources: PodRingResources, kind: string): PodRingData => {
   const deploymentKinds = {
-    Deployment: 'deployments',
-    DeploymentConfig: 'deploymentConfigs',
+    [DeploymentModel.kind]: 'deployments',
+    [DeploymentConfigModel.kind]: 'deploymentConfigs',
   };
```

Comment:
I know this wasn't added in your PR but this is a rather silly map being created just to get the plural form of the model. Which mind you is a hard coded string instead of the model's actual plural value.

Reply:
Yeah, I thought the same thing. I was going to suggest the trimming of this, but it's "a key" to the prop the firehose produces. I suspect in order to refactor the prop values, you'd need to make sure you updated `TransformResourceData` properly, and anyone who uses it - which feels like a large refactor.

Reply:
@christianvogt these are not the plurals from Models but the key that firehose produces to access the data. We could use plurals here but as @andrewballantyne said that would be another refactor.